src/solver/conv_hip_implicit_gemm_v4.cpp

```diff
@@ -1,478 +0,0 @@
-}
-
-bool ConvHipImplicitGemmV4Fwd::IsValidPerformanceConfig(const ConvolutionContext& ctx,
-                                                        const PerformanceImplicitGemm& c) const
-{
-    MIOPEN_LOG_I("");
-    return c.IsValidValue() && c.IsValid(ctx);
-}
-
-bool ConvHipImplicitGemmV4WrW::IsValidPerformanceConfig(const ConvolutionContext& ctx,
-                                                        const PerformanceImplicitGemm& c) const
-{
-    MIOPEN_LOG_I("");
-    return c.IsValidValue() && c.IsValid(ctx);
-}
-
-bool ConvHipImplicitGemmV4_1x1::IsValidPerformanceConfig(const ConvolutionContext& ctx,
-                                                         const PerformanceImplicitGemm& c) const
-{
-    MIOPEN_LOG_I("");
-    return c.IsValidValue() && c.IsValid(ctx);
-}
-
-using ImplicitGemmKernel_t = enum {
-    ImplicitGemmV4     = 0,
-    ImplicitGemmV4_1x1 = 1,
-};
-
-static inline ConvSolution GetSolutionBase(const ConvolutionContext& ctx,
-                                           const PerformanceImplicitGemm& config,
-                                           const ImplicitGemmKernel_t kernel,
-                                           const int n,
-                                           const int k,
-                                           const int ho,
-                                           const int wo)
-{
-    ConvSolution result;
-    KernelInfo construction_parameters;
-
-    const int N1 = config.GemmNRepeat;
-    const int N2 = config.GemmNPerThreadSubC;
-
-    std::size_t b = (static_cast<std::size_t>(n) * ho * wo) / (static_cast<std::size_t>(N1) * N2);
-
-    std::size_t BPerBlock = config.BPerBlock;
-    std::size_t KPerBlock = config.KPerBlock;
-    std::size_t EPerBlock = config.EPerBlock;
-
-    const int ThreadPerLevel1Cluster = config.GemmMLevel0Cluster * config.GemmNLevel0Cluster *
-                                       config.GemmMLevel1Cluster * config.GemmNLevel1Cluster;
-
-    const int block_size = ThreadPerLevel1Cluster;
-
-    std::size_t grid_size = (b / BPerBlock) * (k / KPerBlock);
-
-    std::size_t lkl_wk0 = block_size;
-    std::size_t lkl_wk1 = 1;
-    std::size_t lkl_wk2 = 1;
-
-    construction_parameters.l_wk.push_back(lkl_wk0);
-    construction_parameters.l_wk.push_back(lkl_wk1);
-    construction_parameters.l_wk.push_back(lkl_wk2);
-
-    std::size_t gbl_wk0 = lkl_wk0 * grid_size;
-    std::size_t gbl_wk1 = 1;
-    std::size_t gbl_wk2 = 1;
-
-    construction_parameters.g_wk.push_back(gbl_wk0);
-    construction_parameters.g_wk.push_back(gbl_wk1);
-    construction_parameters.g_wk.push_back(gbl_wk2);
-
-    if(kernel == ImplicitGemmV4)
-    {
-        construction_parameters.kernel_file =
-            "gridwise_convolution_implicit_gemm_v4_nchw_kcyx_nkhw_lds_double_buffer.cpp";
-
-        construction_parameters.kernel_name =
-            "gridwise_convolution_implicit_gemm_v4_nchw_kcyx_nkhw_lds_double_buffer";
```

Comment:
Please find names of removed kernels in `MakeImplGemmDataInvokerFactory()` and remove them there.

Reply:
@atamazov The kernel wrappers mentioned in this comment have been deleted.

Reply:
The invoker logic has been [removed as well](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/274/files#r447191052).